pom.xml

```diff
@@ -35,32 +35,33 @@
         <java.version>1.8</java.version>
 
         <!-- Versions below are for several dependencies we're using in the data & ingest modules -->
-
         <!-- Compile dependencies -->
         <slf4j.version>1.8.0-beta4</slf4j.version>
-        <commons-lang3.version>3.9</commons-lang3.version>
-        <msal4j.version>1.8.1</msal4j.version>
-        <httpclient.version>4.5.8</httpclient.version>
-        <httpcore.version>4.4.11</httpcore.version>
-        <json.version>20190722</json.version>
-        <fasterxml.jackson.core.version>[2.11.0,2.12.0)</fasterxml.jackson.core.version>
-        <azure-storage.version>8.3.0</azure-storage.version>
+        <commons-lang3.version>3.11</commons-lang3.version>
+        <msal4j.version>1.9.1</msal4j.version>
+        <httpclient.version>4.5.13</httpclient.version>
+        <httpcore.version>4.4.13</httpcore.version>
+        <json.version>20201115</json.version>
+        <fasterxml.jackson.core.version>2.12.1</fasterxml.jackson.core.version>
```

Comment:
Previously was [2.11.0,2.12.0) - why?

Reply:
i dont know but i do know that all these jackson libraries are a big mess of versions